test/apps/gups.c

```diff
@@ -195,64 +195,64 @@ int SHMEMRandomAccess(void);
 
```

Comment:
Yeah, I know how much you like two spaces.  :)  For the tests, I mostly don't care, as long as the style is consistent.  Since there are multiple copies of GUPS floating around; it seemed like a good idea to limit the amount of unnecessary code churn.